lib/file-dependencies.rb

```diff
@@ -23,8 +23,8 @@ def process_downloads(files, target, tmpdir)
       full_target = file['target'] ? ::File.join(target, file['target']) : target
       download = FileDependencies::File.fetch_file(file['url'], file['sha1'], tmpdir)
       if (res = download.match(/(\S+?)(\.tar\.gz|\.tgz)$/))
-        prefix = res.captures.first.gsub("#{tmpdir}/", '')
         FileDependencies::Archive.untar(download) do |entry|
+          prefix = file['include_tar_prefix'] ? '' : ::File.join(entry.full_name.split("/").first, '')
```

Comment:
I indeed wanted to get a '/' to the end indeed.
didn't think of using the filesystem separator ( not even sure if this will run on windows :p )